.github/README.md

```diff
@@ -7,9 +7,12 @@ interacts with The Pay's REST API. To get started see examples below.
 
 - PHP 7.4+
 
-- **curl** extension
 - **json** extension
```

Comment:
I just wonder about usefulness of json extension description. We might remove json with curl comment, 'cause it's in composer.json and this could be sufficient.

Reply:
Hmm i edited requirements readme for backward compatibility with SDK instalation without composer.
We use json functions directly so without extension SDK will not work.
Is sposibble than developers stop installing SDK by hand, i hope than noone does, because now we have more than one dependanci and with sugested packages manual instalation will be nightmare 😈.
Is possible rewrite `Requirements` article only as link into composer.json and describe that requirements are here, we will avoid synchronization maintenance between composer.json and readme.md.

Reply:
Good idea, add a link to composer.json for requirements and that's enough.